### PR TITLE
feat: cart management

### PR DESCRIPTION
**Pull request by AmishMishra11**
Added cart management page
-> with an order summary
->price details

Screenshot (20)

### CODE REVIEW

**Review by ishapatell1 — APPROVED** [review]
Amish this looks good enough to merge.

File: cart-management/cart.css [hunk 1]
```diff
@@ -0,0 +1,156 @@
+.container-nav {
+  padding: 0;
+  grid-area: nav;
+}
+.logo {
+  height: 75px;
+  width: 75px;
```
Comment (amarphule) [inline cart-management/cart.css]: you could convert` px `into `rem`

File: cart-management/cart.html [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+              src="/images/E-Cart-Logo-transparent.png"
+              alt=""
+            />
+            <a class="brand-link" href="/index.html">
+              <h2 class="name">Mercart</h2>
+            </a>
+          </div>
+
+          <div class="search">
+            <input type="text" placeholder="waht are you looking for" />
```
Comment (amarphule) [inline cart-management/cart.html]: the typo mistake in placeholder correct it with **_What are you looking for?_**

**Review by amarphule — COMMENTED** [review]
Great work. made some comments please check

**Review by amarphule — COMMENTED** [review]
amish
 also please check with quantity input

File: cart-management/cart.css (hunk 1 above)
Comment (AmishMishra11) [reply cart-management/cart.css]: Ok i will do that

File: cart-management/cart.html (hunk 2 above)
Comment (AmishMishra11) [reply cart-management/cart.html]: Thank you for pointing it out, ill fix it immediately.

**Comment by AmishMishra11** [comment]
> amish also please check with quantity input

yes ive fixed it

**Pull request merged**